src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -25,7 +25,9 @@ import no.skatteetaten.aurora.boober.service.CantusService
 @Service
 class AzureFeature(
     cantusService: CantusService,
-    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String
+    @Value("\${clinger.sidecar.default.version:0}") val sidecarVersion: String,
+    @Value("\${clinger.sidecar.default.ldapurl:}") val defaultLdapUrl: String,
+    @Value("\${clinger.sidecar.default.jwks:http://login-microsoftonline-com.app2ext.sikker-prod.skead.no/common/discovery/keys}") val defaultAzureJwks: String
```

Comment:
Det slår meg forresten nå at jeg ikke er helt sikker på at dette fungerer slik det er lagt opp til her. Dersom jeg logger på sikker-vdi og logger på med `ao login aup` vil jeg vel logge på et "tilfeldig" cluster (f. eks. prod01). 
Vil korrekt boober (boober på test01) da bli brukt hvis jeg kjører `ao deploy test01/test` ? 
@kris456 @stromland - vet dere?

Reply:
@TomasEng ?

Reply:
Deploy vil alltid bruke boober i samme cluster som applikasjonen skal deployes til. Så test01 vil bli brukt i dette tilfellet.

Reply:
Har sett igjennom koden og det ser fint ut. Men skal denne urlen være lik for alle clustre? Hvis ikke kan default verdien fjernes og legges i AuroraConfig?

Reply:
Det er dette som er grunnen til at vi har PR ;)

Her vet vi fra før hvilke adresser som gjelder for hvert enkelt miljø; vi vet at adressen er spesifikk for om tjenesten kjører i preprod eller prod, om den kjører i sikker eller intern. 
Jeg tenkte dette ville være enklere for teamene å forholde seg til; en default som fungerer for det miljøet du er i. Alternativet er å be dem legge inn dette i alle miljøer:
```
config:
    AZURE_JWT_TO_STS_CONVERTER_DISCOVERY_URL: "http://login-microsoftonline-com.app2ext.sikker-prod.skead.no/common/discovery/keys"
    AZURE_JWT_TO_STS_CONVERTER_LDAP_URL: "ldap://ldap.skead.no:389"
```
Det kan vi også gjøre, hvis det er mer i tråd med måten boober er tenkt å fungere.

Reply:
Jeg tenkte på Boober sin AuroraConfig og ikke applikasjonene som skal bruke azure-featuren. Hvis `clinger.sidecar.default.jwks` ikke skal være lik i alle clustre så synes jeg default verdien bør bort og settes eksplisitt for Boober i hvert cluster.

Reply:
Åh, skjønner. Det skal jeg få til :) (planen var uansett å sette det i ac for boober)